meson.build

```diff
@@ -215,3 +215,5 @@ install_data(['nvmf-autoconnect/udev-rules/70-nvmf-autoconnect.rules',
              install_dir: udevrulesdir)
 install_data(disc,
              install_dir: join_paths(sysconfdir, 'nvme'))
+
+ host_machine.cpu_family()
```

Comment:
Newline at the end of the line.

Reply:
oh.. Actually this code no needed.
it was test code to try check what system use(32bit, 64bit).
But the libdir contain build system lib bit, no need of this. will remove that